src/buffer.c

```diff
@@ -139,14 +139,20 @@ open_buffer(
 #endif
        )
     {
+        int old_msg_silent= msg_silent;
```

Comment:
cvwillegen wrote:

> > @@ -139,14 +139,20 @@ open_buffer(
> >  #endif
> >         )
> >      {
> > +        int old_msg_silent= msg_silent;
> 
> Add a space there before the =

Already done.

-- 
hundred-and-one symptoms of being an internet addict:
44. Your friends no longer send you e-mail...they just log on to your IRC
    channel.

 /// Bram Moolenaar -- Bram@Moolenaar.net -- http://www.Moolenaar.net   \\\
///        sponsor Vim, vote for features -- http://www.Vim.org/sponsor/ \\\
\\\  an exciting new programming language -- http://www.Zimbu.org        ///
 \\\            help me help AIDS victims -- http://ICCF-Holland.org    ///